src/lib.rs

```diff
@@ -64,6 +64,7 @@
 #[cfg(feature = "cargo-all")]
 compile_error!("'--all-features' is not supported; use '--features all' instead");
 
+#[cfg(feature = "read_core")]
```

Comment:
I think this also needs to be enabled for `write_core`.

Reply:
Hmm, I didn't see any uses of `alloc` based on `git grep alloc src/write/`